MapboxCoreNavigation/NavigationService.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+import CoreLocation
+import MapboxDirections
+
+protocol NavigationService: class {
```

Comment:
Is this used anywhere?

Reply:
😅 ah, nope... that's for the next stage of changes. Will remove.